template/macros.jinja

```diff
@@ -52,6 +52,11 @@
       tplroot ~ ':tofs:files_switch',
       default_files_switch
   ) %}
+  {# Lookup files or fallback to source_files parameter #}
```

Comment:
Please use `{#-` instead of `{#`, so that it doesn't end up rendering a blank line:

```sls
    - source:

      - salt://template/files/ABC/example.tmpl
      - salt://template/files/ABC/example.tmpl.jinja
      - salt://template/files/Debian/example.tmpl
      - salt://template/files/Debian/example.tmpl.jinja
      - salt://template/files/default/example.tmpl
      - salt://template/files/default/example.tmpl.jinja
```

Reply:
Thanks.